core/message.go

```diff
@@ -73,12 +73,12 @@ func (meta MetaData) SetValue(key string, value []byte) {
 }
 
 // GetValue returns a meta data value by key
-func (meta MetaData) GetValue(key string, defaultValue []byte) []byte {
+func (meta MetaData) GetValue(key string) []byte {
 	if value, isSet := meta[key]; isSet {
 		return value
 	}
 
-	return defaultValue
+	return []byte{}
```

Comment:
We decided to keep it as is. Reasons:
- Currently we don't need to check for existence
- We DO need to check for empty values anyways
- If we every need to check for existence a GetValueOpt function could easily be implemented